### PR TITLE
Samplingfeaturedataset dev

### PR DESCRIPTION
**Pull request by Elijahwalkerwest**
Removed some debugging statements. good to merge now.

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+## ODM2 RESTful Web Services -- Deployment Guide
```
Comment (lsetiawan) [inline DEPLOYMENT.md]: I think this should go in a folder called docs.

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+A Python RESTful web service inteface for accessing data in an ODM2 database via Django rest swagger APIs.
+
+NOTE: Currently this repository is under heavy development, a working version is not guaranteed at any point. We will start creating tagged releases soon; once available, please use the latest tagged release for stability.
+
```
Comment (lsetiawan) [inline DEPLOYMENT.md]: Please remove this whole section, it's already in README.

**Comment by Elijahwalkerwest** [comment]
Should we move anything else to docs?

File: DEPLOYMENT.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+## Production Installation:
```
Comment (lsetiawan) [inline DEPLOYMENT.md]: Top title should be `# ODM2 Restful Web Services -- Production Deployment Guide`

File: DEPLOYMENT.md (hunk 1 above)
Comment (lsetiawan) [reply DEPLOYMENT.md]: Also, rename the file to something more specific like, `production_deployment.md`

File: DEPLOYMENT.md (hunk 3 above)
Comment (Elijahwalkerwest) [reply DEPLOYMENT.md]: Got it.

File: DEPLOYMENT.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+    ```bash
+    $ sudo systemctl restart nginx
+    ```
+
+
+
+
+
+
```
Comment (lsetiawan) [inline DEPLOYMENT.md]: You can just remove everything below here.

File: DEPLOYMENT.md (hunk 1 above)
Comment (Elijahwalkerwest) [reply DEPLOYMENT.md]: Done, should we move anything else to the docs folder?

File: odm2rest/static/js/swagger.json [hunk 5]
```diff
@@ -4,8 +4,8 @@
     "version": "v1",
     "description": "This is a development version for ODM2 REST API",
     "contact": {
-      "email": "landungs@uw.edu",
```
Comment (lsetiawan) [inline odm2rest/static/js/swagger.json]: Eventually we'd want to be able to specify these info within the settings.

File: DEPLOYMENT.md (hunk 1 above)
Comment (lsetiawan) [reply DEPLOYMENT.md]: Nope.

Comment (lsetiawan) [reply DEPLOYMENT.md]: Thanks!

**Comment by lsetiawan** [comment]
@Elijahwalkerwest Another thing, a good workflow is to always try to link a PR to some issue if there is one in this case, this PR Fixes #67. Thanks!

**Comment by Elijahwalkerwest** [comment]
Requested changes made.

**Pull request merged**